ocfweb/docs/docs/staff/staffvm/ocfweb.md

```diff
@@ -0,0 +1,26 @@
+[[!meta title="Running ocfweb on your Staff VM"]]
+
+If you want to work on [`ocfweb`][0] on your Staff VM, it isn't as
+simple as cloning the repo and running `make dev`, unfortunately.
+In order to do so, you will need to add the `ocfweb` dev config to
+your Staff VM's Puppet configuration, so Puppet can install
+`ocfweb`'s dependencies. To do this, clone the [puppet repo][1] or
```

Comment:
> Like Celery getting uninstalled every now and then and needing to be manually reinstalled?

this was because we switched to my fork which supports ssl for backends, and pip doesn't understand the order to uninstall and install packages which provide the same files